src/components/UpgradeTableOfContents.tsx

```diff
@@ -1,10 +1,9 @@
-import { Box, List, ListItem } from "@chakra-ui/react"
-
 import type { ToCItem } from "@/lib/types"
 
-import { BaseLink } from "@/components/Link"
 import { ItemsListProps } from "@/components/TableOfContents/ItemsList"
```

Comment:
Assuming this will be done as part of ToC update

Reply:
Correct!